Gemfile.lock

```diff
@@ -54,6 +55,10 @@ GEM
       unicode-display_width (~> 1.0, >= 1.0.1)
     ruby-progressbar (1.8.1)
     sass (3.4.22)
```

Comment:
sass is installed at 3.4 here, but 3.3 under sassc?  What is the current supported version before this PR, 3.?